lib/routes/dhu/xxgk/news.ts

```diff
@@ -25,25 +26,49 @@ export const route: Route = {
 };
 
 async function handler() {
-    const link = base_url;
-    const response = await got({
-        method: 'get',
-        url: link,
-        headers: {
-            Referer: base_url,
-        },
-    });
-
-    const $ = load(response.data);
+    const link = baseUrl;
+    const { data: response } = await got(link);
+
+    const $ = load(response);
+
+    const items = await Promise.all(
+        $('.cols_list > li')
+            .toArray()
+            .map(async (item) => {
+                item = $(item);
+                const colsTitle = item.find('.cols_title > a');
+                const colsMeta = item.find('.cols_meta');
+
+                // article meta
+                const link = colsTitle.attr('href');
+                const title = colsTitle.text();
+                const pubDate = parseDate(colsMeta.text(), 'YYYY-MM-DD', 'zh-cn');
+
+                // article content
+                const url = `${siteUrl}${link}`;
+                const description = await cache.tryGet(url, async () => {
```

Comment:
Please cache the whole returned object
https://github.com/DIYgod/RSSHub/blob/eb82799227eb01f8cc255aba2387757b733e9cbf/lib/routes/dhu/xxgk/news.ts#L60-L65